### PR TITLE
ZEN-26355 Option to force single column graphs

### PR DESCRIPTION
**Pull request by carlcamera**
Added new user setting to specify user choice for number of columns desired when viewing graphs. Choices are: Auto,1, 2,3. Auto is current (responsive) behavior. 1,2,&3 force layout of graphs panel to the selected number of columns.

### CODE REVIEW

File: Products/ZenUI3/browser/resources/js/zenoss/devdetail.js [hunk 1]
```diff
@@ -436,28 +436,33 @@ var dev_admin = Ext.create('Zenoss.devicemanagement.Administration', {
 
 // find out how many columns the graph panel should be based on
 // on the width of the detail panel
-var center_panel_width = Ext.getCmp('center_panel').getEl().getWidth() - 275,
-    extra_column_threshold = 1000;
+var center_panel_width = Ext.getCmp('center_panel').getEl().getWidth() - 275;
+var extra_column_threshold = 1000;
+var userColumns = Zenoss.settings.graphColumns;
+// graphColumns 0 is "Auto"
+userColumns = ((Zenoss.settings.graphColumns === 0) && (center_panel_width > extra_column_threshold)) ? 2 : 1;
```
Comment (rDr4g0n) [inline Products/ZenUI3/browser/resources/js/zenoss/devdetail.js]: does this work if number of columns is set to 3?

Comment (carlcamera) [reply Products/ZenUI3/browser/resources/js/zenoss/devdetail.js]: absolutely.

Comment (carlcamera) [reply Products/ZenUI3/browser/resources/js/zenoss/devdetail.js]: Line +441 sets userColumns to the settings value

Comment (rDr4g0n) [reply Products/ZenUI3/browser/resources/js/zenoss/devdetail.js]: im parsing js with my eyeballs, but im pretty sure line 443 will always evaluate, and always return either 2 or 1.

Comment (carlcamera) [reply Products/ZenUI3/browser/resources/js/zenoss/devdetail.js]: only when `Zenoss.settings.graphColumns === 0`  that's the "Auto" check

**Pull request merged**